### PR TITLE
Add option for max swap off-chain routing fee

### LINKED ISSUE #100: Add max offchain fee parameter to loop out command
When I loop out, it sets a default max offchain fee, but I'd like to be able to configure this value myself.

### PR DESCRIPTION
**Pull request by Free-FreeThinker**
Add a new command line option `max_swap_routing_fee` for specifying max swap routing fee.

Fixes #100

### CODE REVIEW

File: cmd/loop/loopout.go [hunk 1]
```diff
@@ -115,6 +121,10 @@ func loopOut(ctx *cli.Context) error {
 	}
 
 	limits := getLimits(amt, quote)
+	// If configured, use the specified maximum swap routing fee.
+	if ctx.IsSet("max_swap_routing_fee") {
+		*limits.maxSwapRoutingFee = btcutil.Amount(ctx.Int64("max_swap_routing_fee"))
```
Comment (joostjager) [inline cmd/loop/loopout.go]: Max line len of 80 chars exceeded. We use tab size 8.

Comment (Free-FreeThinker) [reply cmd/loop/loopout.go]: @joostjager, do we have any tool for code formatting?

Comment (guggero) [reply cmd/loop/loopout.go]: There is the `make fmt` command that makes sure that the code is formatted in the way the linter expects.
But long lines aren't automatically wrapped. See [the contribution guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#48-additional-style-constraints-on-top-of-gofmt) to see how we wrap method calls.

This example would look like this:

```
 		*limits.maxSwapRoutingFee = btcutil.Amount(
 		 		ctx.Int64("max_swap_routing_fee"),
 		) 
```

Comment (Free-FreeThinker) [reply cmd/loop/loopout.go]: Thanks @guggero.

CR feedbacks have been addressed, PTAL, thx!

cc @joostjager

**Pull request merged**